src/zope/app/apidoc/codemodule/module.py

```diff
@@ -187,7 +187,9 @@ def get(self, key, default=None):
         obj = safe_import(path)
 
         if obj is not None:
-            return Module(self, key, obj)
+            child = Module(self, key, obj)
+            self._children[key] = child
```

Comment:
Yes. Whether or not this worked before was entirely dependent on what modules were available when the cache was set up.